support/client/lib/vwf/view/mil-sym.js

```diff
@@ -434,4 +435,18 @@ define( [ "module", "vwf/view", "mil-sym/cws", "jquery" ], function( module, vie
         return _rendererReady;
     }
 
+    function getDefaultSymbolFillColor( symbolID, affiliation ) {
+          var color = '000000';                  
```

Comment:
No need to set this here since you set it again below.  In fact, you could return `colors[ affliation ]` and remove the variable entirely.  :smile: 

Also, a friendly reminder to use `const` and `let` (I think `const` is the right option here) rather than `var`.

I have a feeling that you're going to catch me on that one, too.  :wink:

Reply:
That's why I put the TODO comment in about using symbolID in the future.  I figure that there will be certain symbol types (that we could look up by symbolID) that use special colors per affiliation, namely the single point mission graphics that always come up green.  In that case, I'll add a switch statement to segregate the special cases from the standard symbol cases.

Reply:
Gotcha.  Makes sense!  :+1: